package.json

```diff
@@ -3,11 +3,15 @@
   "version": "10.8.0-beta.1",
   "description": "Format whitespace in a SQL query to make it more readable",
   "license": "MIT",
-  "main": "dist/sql-formatter.min.js",
+  "main": "dist/sql-formatter.min.cjs",
   "module": "lib/index.js",
   "types": "lib/src/index.d.ts",
   "exports": {
-    ".": "./lib/index.js"
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./dist/sql-formatter.min.cjs",
+      "types": "./lib/src/index.d.ts"
+    }
   },
   "bin": {
     "sql-formatter": "./bin/sql-formatter-cli.js"
```

Comment:
The bin file has not been migrated to ESM, so `.cjs` suffix is still required, and

```diff
- const { format, supportedDialects } = require('../lib/index');
+ const { format, supportedDialects } = require('./dist/sql-formatter.min.cjs');
```

Reply:
Thanks again.

I'm really not good at this ESM conversion :(